src/cpp/shared/game/save.cpp

```diff
@@ -900,9 +914,40 @@ void game::LoadGame(char* filnam, int newGame, int a3) {
       std::map<std::string, mapVariable> mapVariables;
       for (ironfist_save::save_t::mapVariable_const_iterator it = mp->mapVariable().begin();
         it != mp->mapVariable().end(); it++) {
-        mapVariable *mapVar = new mapVariable;
         std::string mapVariableId = it->id().get();
         MapVarType mapVariableType = StringToMapVarType(it->type());
+		if ((mapVariableType == MAPVAR_TYPE_BOOLEAN) && (!mapVariableId.empty())) {
```

Comment:
What's with the indentation discontinuity?

Reply:
This has now been addressed.